lib/postinstall/update-package.js

```diff
@@ -26,6 +27,7 @@ const changes = {
 
 const patchPackage = async (path, root, config) => {
   const pkg = await PackageJson.load(path)
+  config = config || {}
```

Comment:
another total nit, but this might be more readable if the function definition just sets a default (assuming we're not passing an explicit falsey value anywhere that this is working around, anyway)